packages/kit/src/core/prerender/prerender.js

```diff
@@ -231,18 +245,30 @@ export async function prerender() {
 		const headers = Object.fromEntries(response.headers);
 
 		if (config.prerender.crawl && headers['content-type'] === 'text/html') {
-			for (const href of crawl(body.toString())) {
+			const { ids, hrefs } = crawl(body.toString());
+
+			actual_deeplinks.set(decoded, ids);
+
+			for (const href of hrefs) {
 				if (href.startsWith('data:') || href.startsWith('#')) continue;
```

Comment:
Does this mean (edge case) that if you have hash links to the same page that are wrong that they wouldn't get caught by the missing id logic?

Reply:
It does, yeah. Will fix